pkg/timeline/adaptor/controller/timeline.ts

```diff
@@ -93,6 +99,27 @@ export class TimelineController {
       accounts.map((v) => [v.getID(), v]),
     );
 
+    // [header,avatar]
+    const headerAvatarImages = new Map<AccountID, [string, string]>();
+    const followCounts = new Map<
+      AccountID,
+      { following: number; followers: number }
+    >();
+    for (const [id, _] of accountsMap) {
+      const headerRes = await this.accountModule.fetchAccountHeader(id);
+      const header = Result.isOk(headerRes) ? Result.unwrap(headerRes) : '';
+
+      const avatarRes = await this.accountModule.fetchAccountAvatar(id);
+      const avatar = Result.isOk(avatarRes) ? Result.unwrap(avatarRes) : '';
+      headerAvatarImages.set(id, [header, avatar]);
+
+      const followCountRes = await this.accountModule.fetchFollowCount(id);
+      const followCount = Result.isOk(followCountRes)
+        ? Result.unwrap(followCountRes)
+        : { following: 0, followers: 0 };
```

Comment:
```suggestion
      const header = Result.unwrapOr('')(headerRes);

      const avatarRes = await this.accountModule.fetchAccountAvatar(id);
      const avatar = Result.unwrapOr('')(avatarRes);

      const followCountRes = await this.accountModule.fetchFollowCount(id);
      const followCount = Result.unwrapOr({ following: 0, followers: 0 })(followCountRes);
```
こういう取り出しは `Result.unwrapOr` で簡潔に書けると思います.